packages/react/src/table/table-button.tsx

```diff
@@ -0,0 +1,48 @@
+import React, {forwardRef} from "react";
```

Comment:
Thank you for your contribution. But there is no need to add a `Table.Button`.  Just add a prop like `manually:boolean` to `Table` to control whether to manually trigger lazy loading.

Reply:
Okay, sure, I can work on adding it, are there any other criteria to take into account? I suppose the prop should be added to the `Table.Body` component

Reply:
I imagine the new API will be like `<Table.Body bottom={ReactNode} />` 

`bottom` only shows when `Table.Body` scrolls to the bottom. What do you think @jrgarciadev

Reply:
I am waiting for a reply to start working on this